src/components/Schedule/Schedule.tsx

```diff
@@ -94,6 +94,682 @@ class Schedule extends React.Component<ISentinelProps, ISentinelState> {
             conditionSectionActive ? '' : ' main-section-blue'
           }`}
         >
+          <div className={`bx--col-xs-6 main-section`}>
+            <div className="grid-layout">
+              <div className="grid-item span-1 grid-item-1">
+                <input
+                  tabIndex={-1}
+                  data-tile-input={true}
+                  id="tile-id"
+                  className="bx--tile-input"
+                  defaultValue="tile"
+                  type="checkbox"
+                  name="tiles"
+                  title="tile"
+                />
+                <label
+                  htmlFor="tile-id"
+                  aria-label="tile"
+                  className="bx--tile bx--tile--selectable"
+                  data-tile="selectable"
+                  tabIndex={0}
+                >
+                  <div className="bx--tile__checkmark">
+                    <svg
```

Comment:
Another thing we could extract - all the svgs could be in their `.svg` files. I'm not sure if the project is configured to handle that though, so you can try with 1 svg and if it doesn't work then I'll do it later.

Reply:
not 100% sure what do you mean